eagle-external/eagle-cdh/src/aux/eagle-server.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
Do you have to duplicate the eagle-server.sh ? can you fetch it from eagle-assembly/src/main/bin/eagle-server.sh as part of eagle-cdh/assembly.xml ?